### PR TITLE
update release notes for 1.7

### PR DESCRIPTION
**Pull request by krmayankk**
update release notes for 1.7

### CODE REVIEW

**Comment by krmayankk** [comment]
@dchen1107 @idvoretskyi @luxas @enisoc @caesarxuchao

File: CHANGELOG.md [hunk 1]
```diff
@@ -1003,6 +1003,7 @@ Features for this release were tracked via the use of the [kubernetes/features](
 
 #### PodDisruptionBudget
 * [beta] PodDisruptionBudget has a new field MaxUnavailable, which allows users to specify the maximum number of disruptions that can be tolerated during eviction. For more information, see [Pod Disruptions](https://kubernetes.io/docs/concepts/workloads/pods/disruptions/) and [Specifying a Disruption Budget for your Application](https://kubernetes.io/docs/tasks/run-application/configure-pdb/).
+* [beta] Previously, PodDisruptionBudget Controller used selector-based listers to determine which controller is managing a given Pod. Now ControllerRef proposal is implemented across all controllers that PDB looks at, the  Disruption Controller has been changed to instead use the ControllerRef on a Pod (if any) to determine which controller manages that Pod. Without this change, PDB may look at the wrong controller(s), thus making the wrong decision about whether to allow an eviction of a pod or not.
```
Comment (idvoretskyi) [inline CHANGELOG.md]: Please, remove an indent, together with "[beta]" mentioning.

**Review by idvoretskyi — CHANGES_REQUESTED** [review]

**Comment by enisoc** [comment]
This seems a bit too technical for a user-facing release note. I would make it something like:

```
* PodDisruptionBudget now uses [ControllerRef](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/controller-ref.md) to make the right decisions about Pod eviction even if controllers have overlapping selectors.
```

**Comment by dchen1107** [comment]
+1 on what @enisoc's comment above.

**Comment by krmayankk** [comment]
fixed @dchen1107 @enisoc

**Comment by krmayankk** [comment]
@jbeda can you approve. @enisoc is there any other blocker to get this merged ?

**Comment by enisoc** [comment]
It just needs OWNERS approval.

/assign @dchen1107

**Pull request merged**